forge-gui/res/cardsfolder/upcoming/vindictive_flamestoker.txt

```diff
@@ -0,0 +1,13 @@
+Name:Vindictive Flamestoker
+ManaCost:R
+Types:Creature Phyrexian Wizard
+PT:1/2
+T:Mode$ SpellCast | ValidCard$ Card.nonCreature | ValidActivatingPlayer$ You | Execute$ TrigPutCounter | TriggerZones$ Battlefield | TriggerDescription$ Whenever you cast a noncreature spell, put an oil counter on CARDNAME.
+SVar:TrigPutCounter:DB$ PutCounter | Defined$ Self | CounterType$ OIL | CounterNum$ 1
+A:AB$ Discard | Cost$ 6 R Sac<1/CARDNAME> | Defined$ You | Mode$ Hand | ReduceCost$ X | SubAbility$ DBDraw | SpellDescription$  Discard your hand. then draw four cards. This ability costs {1} less to activate for each oil counter on Vindictive Flamestoker.
+SVar:DBDraw:DB$ Draw | Defined$ You | NumCards$ 4
+SVar:X:Count$CardCounters.OIL
+SVar:BuffedBy:nonCreature
```

Comment:
Not sure if this should be `Card.nonCreature` – I see the version you have here on Deeproot Champion, but am unsure if it works that way

Reply:
Yea won't work
Please fix both